doc/api/deprecations.md

```diff
@@ -719,6 +719,16 @@ The internal `path._makeLong()` was not intended for public use. However,
 userland modules have found it useful. The internal API has been deprecated
 and replaced with an identical, public `path.toNamespacedPath()` method.
 
+<a id="DEP0081"></a>
+### DEP0081: fs.truncate()
+
+Type: Runtime
+
+`fs.truncate()` `fs.truncateSync()` usage with a file descriptor has been deprecated.
+
+*Note*: Please use `fs.ftruncate()` or `fs.ftruncateSync()` 
+if you need to work with file descriptors
```

Comment:
"if you need" is redundant IMO. Also there should be a period at the end of the sentence.